### DISCUSSION TITLE
Vercel Registration from Pakistan

### DISCUSSION THREAD

**Discussion by FaizanJatt** [vercel/community#1196] [Help] [answered]
### Summary

edit=> they fixed my account within 24hrs

I am a recently learning developer on Next Js, spent a while working and learning on some minor projects, just decided to make a clone of facebook for practicing purpose recently and now i realized i am unable to register on vercel.
the following error occurs
Error:The country provided is not allowed. Please contact registration@vercel.com.

Is there any way to get support from vercel about this? 
Or should i try uploading somewhere else for the time being? 
Please suggest me some other place to upload test projects.
Thank you very much.

**Comment by amyegan** [comment, marked as answer]
Sorry you ran into trouble with account registration. 😞 You should email the team at [registration@vercel.com](mailto:registration@vercel.com), and they will help you get your account created. They usually ask about the project you are trying to deploy as one of their manual account validation steps, so it's a good idea to include a short description of your project to speed up the process.

**Reply by FaizanJatt** [reply]
Yes i did write to them about it, I also told them about the project now because you told me to.
Thank you very much

**Reply by msakrambutt** [reply]
Aoa, I face same issue during registration , i am beginner only deploy small program on vercel just for practise, if i mail to registration@vercel.com , and mentioned as a learner or just for practising to deploy small projetct ,vercel team will register myself or not, or they give approval on company.waiting for your positive reply.thanks

**Comment by msakrambutt** [comment]
HI EVERONE, I face  registration issue on vercel to deploy small program, i am beginner only deploy small program on vercel just for practise, if i mail to [registration@vercel.com](mailto:registration@vercel.com) , and mentioned as a learner or just for practising to deploy small projetct ,vercel team will register myself or not, or they give approval on company.waiting for your positive reply.thanks

**Reply by FaizanJatt** [reply]
Yes they fixed my account, msg them and wait for their response

**Comment by msakrambutt** [comment]
Ok thanks
